bots/image-download

```diff
@@ -224,12 +260,16 @@ def prune_images(force, dryrun):
     # iterate over all visible refs (mostly branches)
     # this hinges on being in the top level directory of the the git checkout
     with remember_cwd():
-        os.chdir(os.path.join(BASE, ".."))
+        os.chdir(os.path.join(BOTS, ".."))
         refs = get_refs()
```

Comment:
`get_image_names` should already get all the images from the refs... maybe just add the images from the current files? They might not be checked in.